client/components/header/header.js

```
@@ -61,210 +61,127 @@ export default class Header extends React.Component {
-                    }header-language-btn header-desktop-language-btn header-language-btn-${
-                      lang.slug
-                    }`}
-                    key={lang.slug}
-                    onClick={() => setLanguage(lang.slug)}
-                  >
-                    {lang.text}
-                  </button>
-                ))}
+            {/* Mobile Hamburger */}
+            <div className="header-right header-mobile-only">
+              <div role="button" className="header-hamburger" onClick={this.handleHamburger}>
```

ESLint

[error] A control must be associated with a text label.

(jsx-a11y/control-has-associated-label)